modules/cluster-logging-eventrouter-about.adoc

```diff
@@ -7,6 +7,6 @@
 = About event routing
 
 The Event Router is a pod that watches {product-title} events so they can be collected by OpenShift Logging.
-The Event Router collects events from all projects and writes them to `STDOUT`. Fluentd collects those events and forwards them into the {product-title} Elasticsearch instance. Elasticsearch indexes the events to the `infra` index. 
+The Event Router collects events from all projects and writes them to `STDOUT`. The collector collect those events and forwards them into the {product-title} Elasticsearch instance. Elasticsearch indexes the events to the `infra` index. 
```

Comment:
Verb tense

`The collector collects those events and forwards them into the {product-title} Elasticsearch instance. Elasticsearch indexes the events to the `infra` index.`